webapp/src/shared/components/Game/hooks/useOnlineGame.tsx

```diff
@@ -79,7 +98,14 @@ export function useOnlineGame(gameId: string) {
     }
 
     function handleUpdateGame(info: GameInfoClient) {
+      const now = Date.now();
+      if (updateTimestamp.current) {
+        now - updateTimestamp.current > MAX_CONNECTION_LAG_IN_MS &&
+          warn('slow connection', 'top');
```

Comment:
This is good for now, it may be changed in another ticket to a less invasive warning. That being said I think the smallest problem for a player when is experiencing a laggy connection is to see a notification, he/she cannot play indeed.

Reply:
Last time we played, the warning was also showing from the top, and it was very bothersome (more than just the lag) because I couldn't see the opponent's paddle.
But I understand this may be for a different PR.

Reply:
It really depends also on the screen heigh, I'd say with the score component should be less invasive. But, of course, this can be improved later